packages/address-book-controller/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/address-book-controller`
+
+Manages a list of recipient addresses associated with nicknames.
+
+## Installation
+
+`yarn add @metamask/address-book-controller`
+
+or
+
+`npm install @metamask/address-book-controller`
+
+## Contributing
```

Comment:
I began writing a Usage section for _every single controller_ but it took way too long, so I'll have to add that later. For now I decided to just remove it.